src/app/features/tech-record/components/tech-record-title/tech-record-title.component.ts

```diff
@@ -41,6 +42,14 @@ export class TechRecordTitleComponent implements OnInit {
     return Roles;
   }
 
+  getCompletenessColor(completeness?: string): 'green' | 'red' {
```

Comment:
We should add this to our own technical debt register, Im sure we do something similar for Test results, good opportunity to refactor in the future 😄

Reply:
I do not really get what you mean, but sure, yeah 😊